dev/io.openliberty.data.internal_fat_nosql/fat/src/test/jakarta/data/nosql/FATSuite.java

```diff
@@ -26,6 +29,10 @@
                 DataNoSQLTest.class
 })
 public class FATSuite {
```

Comment:
Add `extends TestContainerSuite` to make sure the remote docker host is set up before this bucket runs.